### ISSUE TITLE
with a dark mode.

### ISSUE THREAD

**Issue by plainee** [surjithctly/mission-control#16]
with a dark mode.

**Comment by mnosov622** [comment]
Hi !

I would love to add dark mode :)

**Comment by surjithctly** [comment]
@mnosov622 it would be great if you could work out something!

**Comment by mnosov622** [comment]
Could you assign this issue to me, please ?

**Comment by mnosov622** [comment]
Thank you !

So I will add button that will toggle dark & light mode, is that okay ?

**Comment by surjithctly** [comment]
Yes that would be perfect. would be nice if it works with @media as well.

**Comment by mnosov622** [comment]
Sorry with what ?

**Comment by surjithctly** [comment]
I mean `prefers-color-scheme`

https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme

**Comment by mnosov622** [comment]
Oh, sure ! Got it :)

**Comment by mnosov622** [comment]
Can you help me, please ?

How can I run the project locally ?

**Comment by plainee** [comment]
I have support Dark mode now, how can I upload it. emm...
